### PR TITLE
[LC-1031] Additional gRPC protos for Consensus Sync

### PR DESCRIPTION
**Pull request by metalg0su**
# Goal
- Add new gRPC protocols (2 messages and 3 protocols)

### CODE REVIEW

File: loopchain/protos/loopchain.proto [hunk 1]
```diff
@@ -50,6 +54,17 @@ message BlockSyncReply {
...
+    optional string peer = 1;
+    optional string channel = 2; // channel ID for multichain network
+}
+
+message HeightResponse {
+    optional string peer = 1;
+    optional string channel = 2; // channel ID for multichain network
+    optional int32 height = 3;
+}
+
```
Comment (hyerim-kim) [inline loopchain/protos/loopchain.proto:67]: Are they enough just to be optional ?

Comment (metalg0su) [reply loopchain/protos/loopchain.proto:68]: They should be required params.

Thank you

File: loopchain/protos/loopchain.proto [hunk 2]
```diff
@@ -18,6 +18,11 @@ service PeerService {
     rpc AddTxList (TxSendList) returns (CommonReply) {}
     // broadcast vote on proposed block
     rpc VoteUnconfirmedBlock (BlockVote) returns (CommonReply) {}
+
+    // exchange height info
+    rpc BlockHeightRequest (HeightRequest) returns (CommonReply) {}
+    rpc BlockHeightResponse (HeightResponse) returns (CommonReply) {}
+    rpc BlockInformationRequest (HeightResponse) returns (CommonReply) {}
```
Comment (windies21) [inline loopchain/protos/loopchain.proto:25]: I think.... BlockRequest is better.

File: loopchain/protos/loopchain.proto [hunk 3]
```diff
@@ -50,6 +55,17 @@ message BlockSyncReply {
     required int32 unconfirmed_block_height = 6;
 }
 
+message HeightRequest {
+    required string peer = 1;
+    required string channel = 2; // channel ID for multichain network
+}
+
+message HeightResponse {
```
Comment (windies21) [inline loopchain/protos/loopchain.proto:63]: How about BlockHeight or Height.

File: loopchain/protos/loopchain.proto (hunk 2 above)
Comment (metalg0su) [reply loopchain/protos/loopchain.proto]: Changed:
- BlockInformationRequest -> BlockRequest

File: loopchain/protos/loopchain.proto (hunk 3 above)
Comment (metalg0su) [reply loopchain/protos/loopchain.proto]: Changed:
- HeightResponse -> PeerHeight

**Pull request merged**